Remove unused mutableOptions variable

LumioseSil opened the pull request:
#### 9b1aa5aeec5d7905aa0c6856abfaa11ca8588d65
<pre>
Remove unused mutableOptions variable
<a href="https://bugs.webkit.org/show_bug.cgi?id=252576">https://bugs.webkit.org/show_bug.cgi?id=252576</a>

Reviewed by Wenson Hsieh and Megan Gardner.

This variable is not used, and clang cannot tell due to all the fancy
smart pointer and function calls going around it, so this variable
should be removed. This variable has not had any use even within the
last 9 years anyway.

*Source\WebCore\editing\cocoa\DictionaryLookup.mm

Canonical link: <a href="https://commits.webkit.org/260570@main">https://commits.webkit.org/260570@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/fe603b0b4d7bc06312e31d1119b4c1897bb22e50

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/108673 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/17774 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/41525 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/117783 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/117979 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/11/builds/112555 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/19225 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/85/builds/9051 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/100906 "Built successfully") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/114440 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/14413 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/97647 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/42383 "Passed tests") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/9/builds/96394 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/29285 "Passed tests") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/34/builds/84119 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/81/builds/10579 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/30635 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/82/builds/11339 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/84/builds/7550 "Passed tests") | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/80/builds/16728 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/50231 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/7308 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/79/builds/12927 "Built successfully") | | | 
| | | | | 
<!--EWS-Status-Bubble-End-->

whsieh reviewed (approved):
Confirmed that this is no longer necessary (nor is it indicative of a bug in our Reveal integration).

LumioseSil committed:
Remove unused mutableOptions variable
https://bugs.webkit.org/show_bug.cgi?id=252576

Reviewed by Wenson Hsieh and Megan Gardner.

This variable is not used, and clang cannot tell due to all the fancy
smart pointer and function calls going around it, so this variable
should be removed. This variable has not had any use even within the
last 9 years anyway.

*Source\WebCore\editing\cocoa\DictionaryLookup.mm

Canonical link: https://commits.webkit.org/260570@main
